TrustModel_Personas.md

```diff
@@ -0,0 +1,197 @@
+# Trust Model Considerations - Personas
+
+## Personas
+Otherwise referred to as actors or threat agents, these are individuals or groups capable of 
+carrying out a particular threat. 
+In identifying personas we consider :
+- The Runtime Environment, Figure 5, Page 19 of CNCF 
+  [Cloud Native Security Paper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/v2/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf).
+  This highlights three layers, Cloud/Environment, Workload Orchestration, Application.
+- The Kubernetes 
+  [Overview of Cloud Native Security](https://kubernetes.io/docs/concepts/security/overview/)
+  identifies the 4C's of Cloud Native 
+  Security as Cloud, Cluster, Container and Code. However data is core to confidential 
```

Comment:
The real difference confidential computing introduces is in-use data. Data at rest and in transit should be protected the usual way (even if we need to work on specific considerations like "how do we provide secure storage to the container").

Where do data at rest and data in transit fit in the existing security model? We should probably mimic that for data in-use.

Reply:
It isn't completely obvious to me yet how we bridge into the existing security model.
In the issue https://github.com/confidential-containers/documentation/issues/28 I try to capture the areas we need to explore specifically for confidential computing (data in use) and the intention in those is to draw on the models already defined within the security whitepaper - lifecycle being a key example. 

I do agree that a guiding consideration needs to be to look to mimic where data at rest and data in transit fit as far as sensible. Was there a specific idea for Personas or more something we need to be mindful off as we further explore content relating to Trust Model?

Reply:
OK.